blocks/api/post.pegjs

```diff
@@ -60,62 +83,52 @@ WP_Block_Void
     return array(
       'blockName'  => $blockName,
       'attrs'      => $attrs,
-      'rawContent' => '',
+      'innerBlocks' => array(),
+      'innerHtml' => '',
     );
     ?> **/
 
     return {
       blockName: blockName,
       attrs: attrs,
-      rawContent: ''
+      innerBlocks: [],
+      innerHtml: ''
     };
   }
 
-WP_Block_Balanced
-  = s:WP_Block_Start ts:(!WP_Block_End c:Any {
-    /** <?php return $c; ?> **/
-    return c;
-  })* e:WP_Block_End & {
-    /** <?php return $s['blockName'] === $e['blockName']; ?> **/
-    return s.blockName === e.blockName;
-  }
+Block_Balanced
+  = s:Block_Start children:(Token / $(!Block_End .))+ e:Block_End
   {
     /** <?php
+    $innerBlocks = array_filter( $children, function( $a ) {
+      return ! is_string( $a );
+    } );
+
+    $innerHtml = array_filter( $children, function( $a ) {
+      return is_string( $a );
```

Comment:
Or here we could probably do `array_filter( $children, 'is_string' );`

Reply:
good catch. I actually intended on maybe creating an `array_partition` function and maybe will still do that. I always get confused which core functions are _functions_ and which are _operators_ (like `empty()`) and cannot be used as `Callable`s

Reply:
changed in 4e41437d